charter/rotating-chair-model.md

```diff
@@ -0,0 +1,57 @@
+
+## Chair Terms and Selection
+
+Each Working Group shall be led by a Chair.
+
+The Chair has a term that is written in the Working Group's own charter. By default, that term is four months.
+
+A person is eligible to chair the Working Group if and only if they satisfy all conditions:
+1. They wish to serve as chair.
+2. They are a member of the Working Group.
+3. They are not the chair of more than one other Working Group.
+    * i.e. One person may chair a maximum of two Working Groups at a time
+4. They have not been the chair of that Working Group more recently than any eligible member.
+    * i.e. Only the eligible members that have least recently (or never) been the chair of that Working Group are able to be chosen.
+
+If there is nobody willing or able to serve as chair, then the Administrative Working Group should be notified for assistance. The AWG can optionally choose to make a delegate from the AWG a temporary chair for the Working Group with the primary goal of reenabling the Working Group's ability to sustain its own chair rotation.
+
+The group will make a reasonable effort to notify potential candidates that a new term is coming so that interested parties can declare their eligibility.
+
+A chair is selected at random from the set of eligible candidates. The candidates and selection are to be documented and made public.
+
+A chair's term does not need to begin at the same time as other working groups.
+
+If a chair steps down, a new one is selected and a new term begins (i.e., interim chairs are not needed)
+
+## Chair Responsibilities
+
+Chairs have responsibilities to their parent Working Group: they ensure that their Working Group is carrying out its mission.
+
+Chairs have responsibility to all participants in transparently reporting the group's activity.
+
+Chairs also have the following regular duties within their Working Groups:
+* Collect agenda items prior to each meeting to ensure a smooth and focused Working Group meeting.
+* Ensure that the Working Group is notified of the agenda for the meeting in a timely manner advance of the meeting.
+* Attend and lead each Working Group meeting.
+    * If the chair is unable to attend a given meeting, they should appoint another member to lead the meeting in their stead.
+* Ensure meeting notes are uploaded to the `governance` repo in the Working Group's respective sub-folder following each meeting.
+* Check into persistent issues present on a given Working Group's repositories of responsibility (ex. bot failures)
+* Ensure that the WG roster list is an accurate reflection of current member activity.
+
+In addition to these responsibilities, a chair may have additional responsibilities specific to their Working Group.
+
+## Is the Rotating Chair Model Right For My Working Group?
+
+(This section is non-normative.)
+
+The following advice and considerations can help a new Working Group decide whether the Rotating Chair Model is a good fit:
+
+* 
```

Comment:
?

Reply:
This part was still WIP! Feel free to look at the new text here.